analysis/create_project_actions.R

```diff
@@ -376,7 +437,12 @@ actions_list <- splice(
     )
   ),
   
-  
+   splice(
+    unlist(lapply(cohorts, 
+                  function(x) generate_ac_te_data(cohort = x)), 
+           recursive = FALSE
+    )
+  ),
   ## Preprocess data -----------------------------------------------------------
```

Comment:
You might want to reorder the actions so the `generate_ac_te_data_{cohort}` type of actions come after `generate_hospitalised_data_{cohort}` type of actions since the latter is needed in the former.